lib/App.js

```diff
@@ -859,6 +874,9 @@ $ sudo systemctl restart docker
     const hasNodeModules = fs.existsSync(path.join(this.path, 'node_modules'));
     const hasPackageJSON = fs.existsSync(path.join(this.path, 'package.json'));
 
+    fse.ensureDirSync(path.join(this._homeyBuildPath, 'node_modules'));
+
+    // TODO support this?
```

Comment:
What is not supported here exactly?

Reply:
I ment its odd to have a node_modules folder without a package.json